src/vue/components/word-usage-examples/word-usage-examples.vue

```diff
@@ -7,7 +7,7 @@
     <div class="alpheios-word-usage-header" data-alpheios-ignore="all">
 
       <word-usage-examples-filters
-        :collapsedHeader = "collapsedHeader"
+        :collapsedHeader = "finalCollapsedHeader"
```

Comment:
This change (and all similiar) are from this PR https://github.com/alpheios-project/components/pull/666